data/alteredAccount/alteredAccount.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package proto;
+
+option go_package = "alteredAccount";
+option (gogoproto.stable_marshaler_all) = true;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+message AlteredAccount {
```

Comment:
comparing this file with other files (`transaction.proto` and `log.proto`) we are missing indentation and comments for the exported items. Please address them

Reply:
> comments for the exported items.

Not necessarly. You can also check block.proto or `smartContractResult.proto`=> those and many others don't have it. Furthermore, would not add comments for generated code since every generated func is exported anyway.

>we are missing indentation
Formatted it